src/lib/jsonParseV2/parseSubscription.cpp

```diff
@@ -256,9 +265,11 @@ static std::string parseEntitiesVector(ConnectionInfo* ciP, EntityIdVector* eivP
 
       return oe.render(ciP, "");
     }
-    EntityId* eiP         = new EntityId();
```

Comment:
I moved this 'new EntityId' down, to avoid to call 'delete' in every 'return caused by error'